.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules
+.idea
```

Comment:
Maybe this is something you could add to your [system level .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)?